src/wp-includes/script-loader.php

```diff
@@ -2049,7 +2049,7 @@ function wp_style_loader_src( $src, $handle ) {
 		return preg_replace( '#^wp-admin/#', './', $src );
 	}
 
-	if ( 'colors' === $handle ) {
+	if ( 'colors' === $handle && isset( $_wp_admin_css_colors ) && is_array( $_wp_admin_css_colors ) ) {
```

Comment:
Do we need the `isset` here ? Adding the `is_array` check in the condition should be enough to avoid the warning.

Reply:
It's possible in some contexts that the variable has not been set, so checking if a non-existent variable is an array would throw an error (at least I expect it would).